extensions/cornerstone-dicom-sr/package.json

```diff
@@ -36,7 +36,7 @@
     "@ohif/extension-cornerstone": "^3.0.0",
     "@ohif/extension-measurement-tracking": "^3.0.0",
     "@ohif/ui": "^2.0.0",
-    "dcmjs": "^0.29.4",
```

Comment:
Why the new dcmjs version?  The adapters code should support circles via a simple extension on the elliptical roi code.

Reply:
It's because of this: https://github.com/dcmjs-org/dcmjs/pull/341